ci-operator/config/openshift-pipelines/pipelines-service/openshift-pipelines-pipelines-service-main.yaml

```diff
@@ -0,0 +1,35 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```

Comment:
go v1.13 was released in 2019. I think it should stick with newer versions, i.e. 1.17 or 1.18